instructions/js/index.js

```diff
@@ -51,14 +51,34 @@ messageForm.addEventListener('submit', (e) => {
 
    newMessage.appendChild(removeButton)
    messageList.appendChild(newMessage)
-   
-
 
    messageForm.reset()
-})
-
 
+  })
+
+
+//AJAX practice... adding GitHub repositories to my portfolio
+const githubRequest = new XMLHttpRequest();
+//Should I put my callback function here? Do I need a callback function?
+//Open method (as the assigment asks you to do)
+githubRequest.open('GET','https://api.github.com/users/mnca16/repos');
+githubRequest.send();
+githubRequest.addEventListener('load', (event) => {
+let repositories = JSON.parse(githubRequest.response)
+console.log(repositories);
```

Comment:
It is not recommended to push code with logs on it. Remove them after you finish debugging